WordPress/Classes/ViewRelated/Reader/Controllers/ReaderStreamViewController.swift

```diff
@@ -361,6 +358,28 @@ import AutomatticTracks
         setupNotificationsBarButtonItem()
     }
 
+    private func layoutEmptyStateView() {
+        guard let emptyStateView else { return }
+
+        // Calculate visible part of the table view in `self.view` coordinates
+        let y: CGFloat = {
+            if let headerView = tableView.tableHeaderView {
+                return headerView.convert(headerView.frame, to: view).maxY
```

Comment:
It probably doesn't make much difference in reality, but I think it only make sense to use `bounds` here, because `headerView.frame` is not in `headerView`'s coordinate space?

```suggestion
                return headerView.convert(headerView.bounds, to: view).maxY
```

Reply:
Good catch. I wrote it hastily, and what it currently doesn't make a whole lot of space. It should be:

```swift
return tableView.convert(headerView.frame, to: view).maxY
```

Convert the header view frame from the table view coordinate space (its superview) to the coordinate space of the main view. 

I updated it.